src/libraries/System.Private.Runtime.InteropServices.JavaScript/tests/System/Runtime/InteropServices/JavaScript/Simple/TimerTests.cs

```diff
@@ -0,0 +1,232 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Runtime.InteropServices.JavaScript.Tests
+{
+    public static class TimerTests
+    {
+        static Function _wrapJsSetTimer = new Function(@"
+            globalThis.setCounter = 0;
```

Comment:
Could we move these javascript chunks to a `.js` file? Just for better readability and maintainability...

Reply:
done